src/Agent.php

```diff
@@ -881,17 +881,31 @@ public static function cronCleanoldagents($task = null)
                         break;
                     case Conf::STALE_AGENT_ACTION_STATUS:
                         if (isset($config['stale_agents_status']) && $item !== null) {
-                            //change status of agents linked assets
-                            $input = [
-                                'id'        => $item->fields['id'],
-                                'states_id' => $config['stale_agents_status'],
-                                'is_dynamic' => 1
-                            ];
-                            if ($item->update($input)) {
-                                $task->addVolume(1);
-                                $total++;
+                            $shouldUpdate = false;
```

Comment:
By convention GLPI use snake case for its variables.

It is not always enforced and you don't really need to fix it here but keep it in mind for futures PR as we try to keep things consistant everywhere if possible.

Reply:
fixed